### PR TITLE
chore: merge RD3 with FAIR Data Hub

### PR DESCRIPTION
**Pull request by joerivandervelde**
The RD3 data model has been (partially) merged with FAIR Data Hub for further increased interoperability and working towards a common datamodel foundation for all *omics apps. Reusing of existing ontologies has resulted in removal of some RD3 ontologies.

### CODE REVIEW

File: data/fairdatahub/RD3.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: RD3
+description: The Rare Disease Data about Data (RD3) sample tracker for Solve-RD
```
Comment (davidruvolo51) [inline data/fairdatahub/RD3.yaml:3]: A **very** minor edit. The "official" title is `Rare Disease Data about Data (RD3) database for Solve-RD`.

File: data/fairdatahub/rd3/molgenis.csv
Comment (davidruvolo51) [inline data/fairdatahub/rd3/molgenis.csv:1]: Excellent! It's nice to see the RD3 table `subjectinfo` joined with `Individuals`. It makes a lot of sense to combine this information.

**Review by davidruvolo51 — DISMISSED** [review]
I have only one very small revision 😄

File: data/fairdatahub/RD3.yaml (hunk 1 above)
Comment (joerivandervelde) [reply data/fairdatahub/RD3.yaml]: Fixed! thanks for the suggestion :)

File: backend/molgenis-emx2-beacon-v2/src/test/java/org/molgenis/emx2/beaconv2/Beaconv2_ModelEndpointsTest.java [hunk 2]
```diff
@@ -48,7 +48,7 @@ void testFilteringTerms() throws Exception {
     assertTrue(json.contains("\"type\" : \"ontology\","));
     assertTrue(json.contains("\"id\" : \"NCIT:C124261\","));
     assertTrue(json.contains("\"label\" : \"Whole Transcriptome Sequencing\","));
-    assertTrue(json.contains("\"scope\" : \"sequencingRuns\""));
```
Comment (mswertz) [inline backend/molgenis-emx2-beacon-v2/src/test/java/org/molgenis/emx2/beaconv2/Beaconv2_ModelEndpointsTest.java:51]: don't you prefer the more specific 'sequencingRuns' ?

File: data/fairdatahub/fairdatapoint/molgenis.csv [hunk 3]
```diff
@@ -22,8 +22,8 @@ Dataset,,temporal,string,,,,,,,,http://purl.org/dc/terms/temporal,The temporal p
 Dataset,,temporalResolution,string,,,,,,,,http://purl.org/dc/terms/temporalResolution,Minimum time period resolvable in the dataset.,DCAT
 Dataset,,wasGeneratedBy,string,,,,,,,,http://www.w3.org/ns/prov#wasGeneratedBy,"An activity that generated, or provides the business context for, the creation of the dataset.",DCAT
 Dataset,,accessRights,string,,,,,,,,http://purl.org/dc/terms/accessRights,Information about who can access the resource or an indication of its security status.,DCAT
-Dataset,,contactPoint,ref,,,,Persons,,,,http://purl.org/dc/terms/contactPoint,Relevant contact information for the cataloged resource. Use of vCard is recommended [VCARD-RDF].,DCAT
-Dataset,,creator,ref_array,,,,Persons,,,,http://purl.org/dc/terms/creator,The entity responsible for producing the resource.,DCAT
+Dataset,,contactPoint,ref,,,,ContactPersons,,,,http://purl.org/dc/terms/contactPoint,Relevant contact information for the cataloged resource. Use of vCard is recommended [VCARD-RDF].,DCAT
```
Comment (mswertz) [inline data/fairdatahub/fairdatapoint/molgenis.csv:25]: later you will see Cohorts simply calls this 'Contacts'

File: backend/molgenis-emx2-beacon-v2/src/test/java/org/molgenis/emx2/beaconv2/Beaconv2_ModelEndpointsTest.java (hunk 2 above)
Comment (joerivandervelde) [reply backend/molgenis-emx2-beacon-v2/src/test/java/org/molgenis/emx2/beaconv2/Beaconv2_ModelEndpointsTest.java:51]: Yes, which is also mentioned in the model itself. But everything else is has Beacon v2 compatible nomenclature now, so renaming this back to "Runs" saves us a headache until we have a good way to perform table/attribute mapping from the base model to Beacon v2.

**Pull request merged**